### ISSUE TITLE
Avoid silent fail when ENV var points to invalid location

### ISSUE THREAD

**Issue by IanMayo** [debrief/pepys-import#459]
A user is working from a laptop.

It has `PEPYS_CONFIG_FILE` pointing to a network location.  But, when disconnected, Pepys-Admin is silently failing.

I was able to reproduce the error by editing the environment variable to point to a file that wasn't present.

I _expect_ `Pepys-Import` to similarly fail silently.  **No** - Pepys admin does report the error. We should use the same string:
```
Successfully set paths for Pepys
Pepys config file not found at location: 'C:\git\pepys-import\default_config.ini2'.
Press any key to continue . . .
```

It would be better if the dialog opened, printed the path that isn't valid, and asked the user to "Press any key" to continue.

**Comment by BarisSari** [comment]
Hello, @IanMayo . This error string comes from `config.py`. So, it is the same for both Pepys-admin and Pepys-import. Would you mind explaining the error a bit more? 

<img width="842" alt="Screen Shot 2020-06-12 at 15 45 12" src="https://user-images.githubusercontent.com/41292272/84503900-b5633780-acc3-11ea-9cdb-8e022a9327a8.png">

**Comment by IanMayo** [comment]
Sure.  On MS-Windows, when we run Pepys-Admin, but the config file is pointing at an invalid location, the command-prompt window briefly opens, reports the error, then closes. This is before the user has had chance to read the error.

So, after reporting the error, we should introduce "Press any key to continue", like we do in Pepys-Import:
![image](https://user-images.githubusercontent.com/1108513/84504400-c7d57500-acb3-11ea-9142-df7a220841cc.png)

**Comment by BarisSari** [comment]
I think `PAUSE` should be added to `pepys_admin.bat` but I can't test it because I don't have a Windows machine. If that's okay, can you solve this issue, @robintw ?

**Comment by IanMayo** [comment]
Oh, if it's just the .bat file I can make the change & fix it.

(not all heroes wear capes!)

**Comment by IanMayo** [comment]
Closed in #463
